Reactions/reactor.cpp

```diff
@@ -0,0 +1,112 @@
+#include<reactor.H>
+
+using namespace amrex;
+
+Array<double, NUM_SPECIES + 1> typVals={-1};
+int eint_rho=1; // in/out = rhoE/rhoY
+int enth_rho=2; // in/out = rhoH/rhoY
+Real relTol=1e-10;
+Real absTol=1e-10;
```

Comment:
brought it back to the old values, reltol at 1e-6, in case pelec CI breaks. LM I think is overriding the defaults all the time. @esclapez can you confirm

Reply:
Indeed, LM always override with its own values defaulted to 1.0e-10, but it doesn't seem to be a problem for CVODE. So LM is good either way.